FF1Blazorizer/Tabs/IncentivesTab.razor

```diff
@@ -122,10 +122,10 @@
 			<EnumDropDown UpdateToolTip="@UpdateToolTipID" Id="kiPlacementMode" IsEnabled="@(Flags.Treasures != false)" TItem="KeyItemPlacementMode" @bind-Value="Flags.KeyItemPlacementMode"></EnumDropDown>
 			<div class="checkbox-cell"></div>
 			<h4>Loose Placement</h4>
-			<EnumDropDown UpdateToolTip="@UpdateToolTipID" Id="loosePlacementMode" IsEnabled="@(Flags.Treasures != false)" TItem="LoosePlacementMode" @bind-Value="Flags.LoosePlacementMode"></EnumDropDown>
-			<CheckBox UpdateToolTip="@UpdateToolTipID" Id="laterLoose" IsEnabled="@(Flags.Treasures != false & Flags.KeyItemPlacementMode != KeyItemPlacementMode.Vanilla)" @bind-Value="Flags.LaterLoose">Later Loose</CheckBox>
-			<CheckBox UpdateToolTip="@UpdateToolTipID" Id="looseNpcBalance" IsEnabled="@(Flags.Treasures != false && (Flags.IncentivizeFreeNPCs != true || Flags.IncentivizeFetchNPCs != true))" @bind-Value="Flags.LooseItemsNpcBalance">Favor NPCs</CheckBox>
-			<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="LooseExcludePlacedDungeonsCheckBox" IsEnabled="@(Flags.Treasures != false)" @bind-Value="Flags.LooseExcludePlacedDungeons">No Loose Items in Incentivized Dungeons</TriStateCheckBox>
+			<EnumDropDown UpdateToolTip="@UpdateToolTipID" Id="loosePlacementMode" IsEnabled="@(Flags.Treasures != false && Flags.Flags.IncentivizedItemCountMax > Flags.Flags.IncentivizedLocationCountMin)" TItem="LoosePlacementMode" @bind-Value="Flags.LoosePlacementMode"></EnumDropDown>
```

Comment:
this doesn't work correctly. Everything that isn't checked on the right side in the Incentives Tab is loose as well.